tests/test_compose_curated_robustness.py

```
@@ -0,0 +1,400 @@
+                    [support_thalamic("clean-1")],
+                )
+                default = source / compose_curated.FFPC_UNITS_MIGRATION
+                default.parent.mkdir(parents=True)
+                if member == "directory":
+                    default.mkdir()
+                else:
+                    default.symlink_to(root / "missing-target.json")
+
+                # The directory hits the new default-calibration guard; a
+                # symlink is already refused by the alias-hardened scanner.
+                with self.assertRaisesRegex(
```

Ruff (0.16.3)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)